### DISCUSSION TITLE
MAC OS BETA 28

### DISCUSSION THREAD

**Discussion by Frankiemagic13** [obsproject/obs-studio#6939] [Development Questions] [answered]
Will beta 28 for mac include application/window audio like windows does?

**Comment by sebastian-s-beckmann** [comment, marked as answer]
The new macOS Screen Capture source does include audio on macOS 13.
In the future, I'd recommend joining our [Discord server](https://obsproject.com/discord) for questions like these.

**Comment by Fenrirthviti** [comment]
This sounds more likely to be a support request, and we are not currently accepting support requests on GitHub Discussions. Make sure you read any [pinned posts](https://github.com/obsproject/obs-studio/discussions/5206) on discussion boards before opening a new discussion. Please use our [forums](https://obsproject.com/forum/) or [Discord](https://obsproject.com/discord/) for further assistance.

Thank you!
